templates/www.conf.j2

```diff
@@ -1,15 +1,15 @@
 [www]
-listen = 127.0.0.1:9000
-listen.allowed_clients = 127.0.0.1
+listen = {{ php_fpm_listen }}
+listen.allowed_clients = {{ php_fpm_listen_allowed_clients }}
 user = {{ php_fpm_pool_user }}
 group = {{ php_fpm_pool_group }}
 
 listen.owner = {{ php_fpm_pool_user }}
 listen.group = {{ php_fpm_pool_group }}
 
-pm = dynamic
-pm.max_children = 50
-pm.start_servers = 5
-pm.min_spare_servers = 5
-pm.max_spare_servers = 5
-pm.max_requests = 500
+pm = {{ php_fpm_pm }}
```

Comment:
I think we should limit this PR's scope. #302 already looks at rearranging the way pools work. This PR should focus on just the JIT variables, IMO.